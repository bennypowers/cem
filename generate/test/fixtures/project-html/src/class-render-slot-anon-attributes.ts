@customElement('class-render-slot-anon-attributes')
class ClassRenderSlotAnonAttributes extends LitElement {
  render() {
    return html`
      <!-- summary: summary
           description: |
             \`<slot>\` -->
      <slot class="foo"></slot>
    `;
  }
}
