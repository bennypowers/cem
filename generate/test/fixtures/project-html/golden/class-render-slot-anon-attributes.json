{
  "schemaVersion": "2.1.0",
  "modules": [
    {
      "kind": "javascript-module",
      "path": "src/class-render-slot-anon-attributes.js",
      "declarations": [
        {
          "name": "ClassRenderSlotAnonAttributes",
          "superclass": {
            "name": "LitElement",
            "package": "lit"
          },
          "source": {
            "href": "https://github.com/bennypowers/cem/tree/main/generate/test/fixtures/project-html/src/class-render-slot-anon-attributes.ts#L1"
          },
          "kind": "class",
          "tagName": "class-render-slot-anon-attributes",
          "slots": [
            {
              "name": "",
              "summary": "summary",
              "description": "`\u003cslot\u003e`"
            }
          ],
          "customElement": true
        }
      ],
      "exports": [
        {
          "kind": "custom-element-definition",
          "name": "class-render-slot-anon-attributes",
          "declaration": {
            "name": "ClassRenderSlotAnonAttributes",
            "module": "src/class-render-slot-anon-attributes.js"
          }
        }
      ]
    }
  ]
}